### PR TITLE
fix: Upload assets from github action

### PR DESCRIPTION
**Pull request by ruanxin**
**Description**

related https://github.com/kyma-project/lifecycle-manager/issues/1811
**Related issue(s)**

### CODE REVIEW

File: scripts/release/upload_assets.sh [hunk 1]
```diff
@@ -31,14 +31,10 @@ MODULE_VERSION=${PULL_BASE_REF} make build-manifests
 echo "Generated template-operator.yaml:"
 cat template-operator.yaml
 
-MODULE_VERSION=${PULL_BASE_REF} make build-module
```
Comment (amritanshusikdar) [inline scripts/release/upload_assets.sh:34]: Why do we need to get rid of module template generation?

File: scripts/release/upload_assets.sh [hunk 2]
```diff
@@ -62,6 +58,4 @@ UPLOAD_URL="https://uploads.github.com/repos/kyma-project/template-operator/rele
 
 echo "$UPLOAD_URL"
 uploadFile "template-operator.yaml" "${UPLOAD_URL}?name=template-operator.yaml"
-uploadFile "module-template.yaml" "${UPLOAD_URL}?name=module-template.yaml"
```
Comment (amritanshusikdar) [inline scripts/release/upload_assets.sh:65]: Same here, as stated in the previous comment

File: scripts/release/upload_assets.sh (hunk 1 above)
Comment (ruanxin) [reply scripts/release/upload_assets.sh:34]: moduletemplate is only for managed use case, which is generated by internal pipeline into kcp, the open source release, we don't need to provide it, you can check other kyma module release as example. https://github.com/kyma-project/keda-manager/releases/tag/1.2.0

Comment (amritanshusikdar) [reply scripts/release/upload_assets.sh:34]: Alright, LGTM :D

**Review by amritanshusikdar — APPROVED** [review]
LGTM

**Pull request merged**